src/Web.cpp

```diff
@@ -1067,6 +1067,10 @@ void explorerHandleDeleteRequest(AsyncWebServerRequest *request) {
 		param = request->getParam("path");
 		convertFilenameToAscii(param->value(), filePath);
 		if (gFSystem.exists(filePath)) {
+			// stop playback, file to delete might be in use
+			if (!gPlayProperties.pausePlay) {
```

Comment:
I assume that if we are paused, delete the file and then resume the playback, that we also get a crash. So here you should rather check, if we are in playback mode (not only if we are not paused). 

For a fast fix, this is ok, but it'd also be more elegant to check, if we are deleting a file/folder from the folder which is currently playing, before stopping the music playback (not sure how complicated this would be to implement).